src/zoid/auth/component.jsx

```diff
@@ -62,93 +72,119 @@ export function getAuthComponent() : ZoidComponent<AuthPropsType> {
                     required:   false
                 },
 
-                clientID: {
-                    type:       'string',
-                    value:      () => getClientID()
+                client_id: {
+                    type:               'string',
+                    queryParam:         true,
+                    value:      () =>   getClientID(),
+                    required:           false
                 },
-        
+
                 sessionID: {
                     type:       'string',
                     value:      getSessionID,
                     queryParam: false
                 },
-        
+
                 authSessionID: {
                     type:       'string',
                     queryParam: false,
                     required:   false
                 },
-                
+                scope: {
+                    type:       'string',
+                    queryParam: true,
+                    required:   false
+                },
+                redirect_uri: {
```

Comment:
I'd like to see if we can remove this `redirect_uri` prop and replace it with the `xprops.getParentDomain()` functionality as the standard way to get the merchant's domain.
```suggestion
```

Reply:
Agreed. Discussed options on the call and listed here: https://github.com/paypal/paypal-sdk-spec/pull/55/files#r721602777

Reply:
we removed the redirecturl.